mail/aerc/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                aerc
+version             0.1.0
+categories          mail
+license             MIT
+maintainers         herrbischoff.com:marcel
```

Comment:
MacPorts would ideally like your GitHub handle to be specified here as well; doing so allows you to be automatically notified of any proposed changes to your port via GitHub.

Additionally, would you be interested in having this port be openmaintainer?